src/middlewared/middlewared/api/v25_04_0/virt.py

```diff
@@ -240,7 +240,7 @@ class VirtInstanceDeviceAddArgs(BaseModel):
 
 
 class VirtInstanceDeviceAddResult(BaseModel):
-    result: dict
+    result: Literal[True]
```

Comment:
Should it be `result: bool` ?

Reply:
Well, it won't ever return False, if there is an error it will be a CallError.
I think I have seen other places using Literal[True] but I can change to bool fi thats preferred

Reply:
Nope, if it raises an exception otherwise, this is fine.